### PR TITLE
[MM-70]: added the test cases for the context menu features and message access request in to-do list.

### PR DESCRIPTION
**Pull request by arush-vashishtha**
###Summary
This PR consists the test cases for the Following scenarios,

- The feature to create gitlab issue from context menu.
- The feature to attach message to gitlab issue.
- The format message access request notification in the to-do list.

### CODE REVIEW

File: data/test-cases/plugins/gitlab/context-menu-feature/Attach_message_gitlab_issue.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+# (Required) Ensure all values are filled up
+name: "Create a Gitlab issue from the context menu on MM."
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/context-menu-feature/Attach_message_gitlab_issue.md]: Is it context menu?

File: data/test-cases/plugins/gitlab/context-menu-feature/Create_gitlab_issue.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Hover cursor over a non-system message and click on the `message-actions` then select `Create Gitlab Issue` on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/context-menu-feature/Create_gitlab_issue.md]: What do you mean by a non-system message, use appropriate term for this.

File: data/test-cases/plugins/gitlab/subscriptions-and-notifications/Todo_list_member_access_request.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Create subscription in any desired channel or DM/Gm on MM to any desired public project in a public group on gitlab.
+2. Create a member access request from a different user to the desired public project on gitlab.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/subscriptions-and-notifications/Todo_list_member_access_request.md]: for which the subscription is created.

File: data/test-cases/plugins/gitlab/context-menu-feature/Create_gitlab_issue.md (hunk 2 above)
Comment (arush-vashishtha) [reply data/test-cases/plugins/gitlab/context-menu-feature/Create_gitlab_issue.md]: By `non-system message` i meant the messages that were added by a users account, it was mentioned in the PR as `non-system message` that why I followed this naming format,  but not i have changed it by `user message`.

https://github.com/mattermost/mattermost-plugin-gitlab/pull/306

File: data/test-cases/plugins/gitlab/subscriptions-and-notifications/Todo_list_member_access_request.md (hunk 3 above)
Comment (arush-vashishtha) [reply data/test-cases/plugins/gitlab/subscriptions-and-notifications/Todo_list_member_access_request.md]: For subscription here I thought that the todo list is connected with subscriptions, as per the discussion with Raghav i have updated the test cases accordingly.

**Review by AayushChaudhary0001 — APPROVED** [review]

File: data/test-cases/plugins/gitlab/context-menu-feature/Attach_message_gitlab_issue.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+# (Required) Ensure all values are filled up
+name: "Add comment to a gitlab issue from the `Attach Message to GitLab Issue` option in the context menu on MM."
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/gitlab/context-menu-feature/Attach_message_gitlab_issue.md]:
```suggestion
name: "Add comment to a gitlab issue from the `Attach to GitLab Issue` option in the context menu on MM."
```

File: data/test-cases/plugins/gitlab/context-menu-feature/Attach_message_gitlab_issue.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Hover cursor over a user message and click on the `message-actions` then select `Attach Message to GitLab Issue` on MM.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/gitlab/context-menu-feature/Attach_message_gitlab_issue.md]:
```suggestion
1. Hover cursor over a user message and click on the `message-actions` then select `Attach to GitLab Issue` on MM.
```

**Review by raghavaggarwal2308 — APPROVED** [review]

**Pull request merged**